packages/client/components/MeetingsDash.tsx

```diff
@@ -62,25 +52,62 @@ const Flash = styled('img')({
 })
 
 const MeetingsDash = (props: Props) => {
-  const {viewer} = props
+  const {meetingsDashRef, viewer} = props
   const teams = viewer?.teams ?? []
-  const activeMeetings = teams
-    .flatMap((team) => team.activeMeetings)
-    .filter(Boolean)
-    .sort((a, b) => (a.createdAt > b.createdAt ? -1 : 1))
-  const hasMeetings = activeMeetings.length > 0
+  const activeMeetings = useMemo(() => {
+    const meetings = teams
+      .flatMap((team) => team.activeMeetings)
+      .filter(Boolean)
+      .sort((a, b) => (a.createdAt > b.createdAt ? -1 : 1))
+    return meetings.map((meeting, displayIdx) => ({
+      ...meeting,
+      key: meeting.id,
+      displayIdx
+    }))
+  }, [teams])
+  const transitioningMeetings = useTransition(activeMeetings)
+  const maybeBigDisplay = useBreakpoint(Breakpoint.BIG_DISPLAY)
   const maybeTabletPlus = useBreakpoint(Breakpoint.FUZZY_TABLET)
-  const maybeBigDisplay = useBreakpoint(1900)
+  const isInit = useInitialRender()
+  const cardsPerRow = useCardsPerRow(meetingsDashRef)
+  const topByRow = useTopPerRow(cardsPerRow, activeMeetings)
+  const hasMeetings = activeMeetings.length > 0
+  const totalRows = !activeMeetings.length ? 0 : Math.ceil(activeMeetings.length / cardsPerRow)
   useDocumentTitle('Meetings | Parabol', 'Meetings')
-  if (!viewer) return null
+  if (!viewer || isInit) return null
```

Comment:
this is a good catch if you refresh the page, but the cards all animate into place when they shouldn't under the following scenario:

1. goto /meetings
2. click the first meeting
3. hit the browser back button
4. see the cards animate when they shouldn't

Reply:
When going back, `cardsPerRow` could be calculated on the third render. On the second render before it was calculated it would return 1, the default value. By changing the default value to null and only showing the meeting cards once it's calculated, I can remove `isInit` and fix this bug